readthedocs/projects/views/private.py

```diff
@@ -113,6 +114,25 @@ def get_context_data(self, **kwargs):
             # Alternatively, dynamically override super()-derived `project_list` context_data
             # context[self.get_context_object_name(filter.qs)] = filter.qs
 
+            projects = AdminPermission.projects(user=self.request.user, admin=True)
+            n_projects = projects.count()
+            if n_projects < 3 and (timezone.now() - projects.first().pub_date) < 7:
```

Comment:
Ah yeah, the project count check is a good addition here too :+1:

Reply:
```suggestion
            if n_projects < 3 and (timezone.now() - projects.first().pub_date).days < 7:
```

I missed the `.days` here.

Reply:
I was going to ask about this, I figured I just learned something new :smile: